superset-frontend/src/components/ListView/Filters/Select.tsx

```diff
@@ -16,137 +16,76 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React, { useState } from 'react';
-import { withTheme, SupersetThemeProps } from '@superset-ui/core';
-import {
-  Select,
-  PaginatedSelect,
-  PartialThemeConfig,
-} from 'src/components/Select';
+import React, { useState, useMemo } from 'react';
+import { withTheme, SupersetThemeProps, t } from '@superset-ui/core';
```

Comment:
It seems theme variables are not used directly by this component anymore. Can we clean up `withTheme`?

On a separate note, I think we should prefer `useTheme` in all future refactoring, too---unless for wrapping legacy class-based components. I added an entry in the [CSS Styled Guide](https://github.com/apache/superset/wiki/Emotion-Styling-Guidelines-and-Best-Practices/_compare/4ea6764f497f352bff4742b3dfa192b4c46d633a...9cd7da713051b2b98eeb512fd92da2b707145c02#diff-5d6646c4ad4792858d153e144bf5fac28133c6005ed3bfd54ccaddde9c22732eR11).

cc. @evans @etr2460 Lmk if you have different opinions.

Reply:
Agreed

Reply:
`withTheme` removed.